lib/vnet/app_resolver.go

```diff
@@ -109,7 +111,7 @@ type TCPAppResolver struct {
 func NewTCPAppResolver(appProvider AppProvider, opts ...tcpAppResolverOption) (*TCPAppResolver, error) {
 	r := &TCPAppResolver{
 		appProvider: appProvider,
-		slog:        slog.With(teleport.ComponentKey, "VNet.AppResolver"),
+		log:         logutils.NewPackageLogger(teleport.ComponentKey, "VNet.AppResolver"),
```

Comment:
Isn't `NewPackageLogger` meant to be used for package-level vars?

Reply:
I forgot that this package even already defines a package-level `log`, I'm going to use it here then.